scss/detail.scss

```diff
@@ -37,97 +36,152 @@ $detail-margin-bottom: 30px;
       position: absolute;
       left: 0;
       top: -2px;
-    }
-
-    @include xl {
-      position: absolute;
-      bottom: -40px;
-      margin: 0 auto;
-    }
-  }
-
-  .card {
-    padding: 25px;
-    display: flex;
-    @include box-shadow;
-
-    .card-body{
-      width: 100%;
-      order: 2;
-
-      .header {
-        margin-bottom: $header-margin-bottom;
-
-        .title {
-          @include bold-text($title-font-size);
-        }
-
-        .subtitle {
-          @include bold-text($subtitle-font-size);
-          color: $nevada-gray;
-        }
-
-        &:after {
-          content: '';
-          display: inline-block;
-          width: 100%;
-          height: 5px;
-          background: $earls-green;
-        }
-
-        @include xl {
-          margin-bottom: $header-margin-bottom * 0.35;
-
-          .title {
-            @include bold-text($title-font-size * 0.8);
-          }
-
-          .subtitle {
-            @include bold-text($subtitle-font-size * 0.65);
-          }
-        }
-      }
-
-      .detail {
-
-        .attribute {
-          margin-bottom: $detail-margin-bottom;
-
-          .label {
-            @include bold-text($attribute-font-size);
-          }
-
-          .value {
-            color: $nevada-gray;
-            font-size: $attribute-font-size;
-          }
-
-          @include xl {
-            margin-bottom: $detail-margin-bottom * 0.7;
-
-            .value {
-              font-size: $attribute-font-size * 0.7;
-            }
-
-            .label {
-              @include bold-text($attribute-font-size * 0.7);
-            }
-          }
-        }
-      }
-    }
+		}
```

Comment:
check indentation on this file

Reply:
hmm it kinda looks like some sort of github visual bug, the file seems fine in my editor.
![image](https://user-images.githubusercontent.com/10158754/80532602-057b7a00-8962-11ea-952e-3bbd38950d36.png)